history/management/commands/pull_balance.py

```diff
@@ -38,15 +38,3 @@ def handle(self, *args, **options):
                                 deposited_amount_btc=deposited_amount_btc if ticker == 'BTC' else 0.00,
                                 deposited_amount_usd=deposited_amount_usd if ticker == 'BTC' else 0.00)
                     b.save()
-
-        for b in Balance.objects.filter(date_str='0'):
-            # django timezone stuff , FML
-            b.date_str = datetime.datetime.strftime(b.created_on - datetime.timedelta(hours=int(7)), '%Y-%m-%d %H:%M')
-            b.save()
-
-        # normalize trade recommendations too.  merp
-        for tr in Trade.objects.filter(created_on_str=''):
-            # django timezone stuff , FML
-            tr.created_on_str = datetime.datetime.strftime(
-                tr.created_on - datetime.timedelta(hours=int(7)), '%Y-%m-%d %H:%M')
-            tr.save()
```

Comment:
Is it correct to remove this? Or do we need a way to distinguish some objects from the others?